apps/stats-web/env/.env.sample

```
@@ -1,8 +1,10 @@
-NEXT_PUBLIC_API_BASE_URL=
-NEXT_PUBLIC_UNLEASH_ENABLE_ALL=true
-NEXT_PUBLIC_UNLEASH_FRONTEND_API_URL=
-NEXT_PUBLIC_UNLEASH_FRONTEND_API_TOKEN=
+VITE_API_BASE_URL=
+VITE_BASE_API_MAINNET_URL=
+VITE_BASE_API_TESTNET_URL=
+VITE_BASE_API_SANDBOX_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_BASE_API_SANDBOX_URL key should go before the VITE_BASE_API_TESTNET_URL key

(UnorderedKey)